pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = ""
 authors = ["James Turk <james@openstates.org>"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
```

Comment:
Because the github actions workflows that execute these jobs all specify python 3.8, e.g.

https://github.com/openstates/people/blob/a337c4b4d86bbb7b1911c0ec4cac0010c092867a/.github/workflows/lint-yaml.yml#L19

https://github.com/openstates/people/blob/main/.github/workflows/test.yml#L34